public/index.html

```diff
@@ -109,13 +109,12 @@
 
     <!-- Status logging -->
     <div class="log">{{ logtext }}</div>
-    <p>{{ identity }}</p>
 
   </script>
 
-  <script src="//media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
+  <script src="//media.twiliocdn.com/sdk/js/client/v1.4/twilio.min.js"></script>
```

Comment:
I recall we agreed on using the Twilio.js client version 1.3. v1.4 is still in beta.
https://www.twilio.com/docs/api/client/changelog-14-beta